exporter/lokiexporter/logproto/README.md

```diff
@@ -0,0 +1,14 @@
+# Notice
+
+This code was copied from https://github.com/grafana/loki/tree/master/pkg/logproto, due to incompatible dependency 
+issues when this exporter was added to the top level binary (top level go.mod replace, even with a specific version,
+is not a good options due to other components' dependency requirements).
+
```

Comment:
Would it be possible to extract this code in a independent module on the loki project in the future to remove this duplicated code and ensure nice dependency?

Reply:
Saw the comment, please file an issue and assign to you so we can track progress on the removing of this code?

Reply:
Absolutely, @bogdandrutu! However, due to my permissions, I'll either need your or someone else to assign the issue to me. I also have a few other issues that need assigned and labelled for tracking.

Reply:
@bogdandrutu - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2386